policyengine_canada/parameters/gov/provinces/nt/tax/income/credits/living_tax_credit/max_amount.yaml

```diff
@@ -0,0 +1,9 @@
+description: Northwest Territories Credit cost of living tax credit maximum aount
```

Comment:
description: Northwest Territories Credit cost of living tax credit maximum amount.

- also add periods in the end of parameter descriptions